src/components/Courses.js

```diff
@@ -336,13 +350,14 @@ class Courses extends React.Component {
                         visibleColumns={visibleColumns}
                         columnWidths={columnWidths}
                         columnMapping={columnMapping}
+                        resultsRowClickHandlers={false}
                         formatter={resultsFormatter}
+                        resultsRowFormatter={DefaultMCLRowFormatter}
```

Comment:
I'm interested to know why this is necessary: isn't it the default
?

Reply:
You are correct, it is the default. Deleting, thank you!